pyuvdata/uvflag.py

```diff
@@ -0,0 +1,832 @@
+# -*- mode: python; coding: utf-8 -*
+# Copyright (c) 2019 Radio Astronomy Software Group
+# Licensed under the 2-clause BSD License
+
+from __future__ import print_function, division, absolute_import
+import numpy as np
+import os
+from . import UVData
+from . import UVCal
+from . import utils as uvutils
+from . import version as uvversion
+from . import telescopes as uvtel
+import warnings
+import h5py
```

Comment:
I think we agreed to move these imports inside the function where they're needed and to decorate tests that require h5py with the `skipIf_no_healpy` decorator.